code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -84,11 +84,11 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 	moles_visible = 1
 	dangerous = TRUE
 
-/datum/gas/brown_gas //This is nitric oxide, but given generic name to avoid confusion with nitrous oxide(N20 vs. NO2)
```

Comment:
fun fact, nitric oxide is NO, not NO2, which is just nitrogen dioxide.
thanks, wikipedia!

accordingly, rename it to that, [since nitryl actually means something](https://en.wikipedia.org/wiki/Nitryl).

Reply:
also relevant fact: nitrogen dioxide is a brown gas, thus why it is called `brown gas`. it lacks the `'s` that would make it brown's gas.

Reply:
The difference between nitrogen dioxide and nitryl is merely one of semantics; they're both NO2. I'm more concerned with preventing confusion with nitrogen and n2o than with having the 100% proper name
As for brown vs Brown's the gas was called Brown's in various places and was originally named that by mistake; brown gas was just an easy fix but far from the best

Reply:
I tried renaming it to nitrogen dioxide in another pr and people said it was shit. Nitryl sounds better and means nitrogen dioxide when inside a carbonic chain.

tl;dr Nitryl is better name